### DISCUSSION TITLE
TailwindCSS Icon

### DISCUSSION THREAD

**Discussion by rogadev** [tailwindlabs/heroicons#671] [Ideas]
I searched "Tailwind" and did not see a TailwindCSS icon. I'd love to tell the world that "this was built with TailwindCSS and TailwindUI" but you don't seem to have an icon for that. Putting it into Fontawesome would also be amazing.

**Comment by zefir-git** [comment]
Why burden Heroicons with brands/logos when https://simpleicons.org/ is specifically for this purpose
